_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -1038,7 +1038,7 @@ Next, use the `account-baseline-root` module from the Gruntwork Infrastructure a
 [source,hcl]
 ----
 module "root_baseline" {
-  source = "git::git@github.com:gruntwork-io/module-security.git//modules/account-baseline-root?ref=v0.36.0"
+  source = "git::ssh://git@github.com:gruntwork-io/terraform-aws-security.git//modules/account-baseline-root?ref=v0.44.10"
```

Comment:
Did any of the input or output vars change across these 8 backwards incompatible versions?

Reply:
Before merging, @marinalimeira could you look into this?

Reply:
There are some new outputs: `organization_root_id`, `aws_ebs_encryption_by_default_enabled` and `aws_ebs_encryption_default_kms_key`

Some variables were added: `iam_password_policy_allow_users_to_change_password`, `iam_password_policy_hard_expiry`, `encrypted_volumes_kms_id`, `rds_storage_encrypted_kms_id`, `iam_role_tags`, `cloudtrail_allow_kms_describe_key_to_external_aws_accounts`, `cloudtrail_tags`, `cloudtrail_is_organization_trail`, plus variables for IAM access analyzer and EBS encryption.

None of these look like a breaking change in the existing interface for this guide. What do you think?

Reply:
Are any of those new variables _required_? Or do they all have reasonable defaults?

Reply:
All of them have a default.

I made these updates while testing the guide by myself. After updating the version number, was not necessary to add any other variable.

Reply:
Roger, thx for checking!